src/data/Pager/PagerTheme.ts

```diff
@@ -2,19 +2,42 @@ export interface PagerTheme {
   base: string;
   pages: string;
   ellipsis: string;
+  page: string;
+  active: string;
+  pagerDisplayItems: string;
+  control: string;
+  firstPage: string;
+  prevPage: string;
+  lastPage: string;
+  nextPage: string;
 }
 
-const baseTheme: Partial<PagerTheme> = {
+const baseTheme: PagerTheme = {
   base: 'items-center flex user-select-none',
   arrows: 'h-8 w-8',
   pages: 'inline-flex',
-  ellipsis: 'cursor-pointer'
+  ellipsis: 'cursor-pointer',
+  page: 'py-1',
```

Comment:
why don't we make these in a sub object?

Reply:
nice catch, I'll update (pages->page)